ci/script.sh

```diff
@@ -9,12 +9,18 @@ cargo build --verbose
 cargo doc --verbose
 
 # Run tests. If we have nightly, then enable our nightly features.
+CARGO_TEST_EXTRA_FLAGS=""
 if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
-  cargo test --verbose --features unstable
-else
-  cargo test --verbose
+  CARGO_TEST_EXTRA_FLAGS="--features unstable"
```

Comment:
Incidentally, I think this whole thing can just be removed now. `unstable` is now a no-op. :-)

Reply:
I removed the features but left the logic in place in case it is ever used again.